src/Controls/Foldable/src/Controls.Foldable.csproj

```diff
@@ -14,7 +14,7 @@
     <!-- NuGet package information -->
     <IsPackable>True</IsPackable>
     <PackageTags>$(DefaultPackageTags);foldable;surface-duo</PackageTags>
-    <Description>Foldable or multi-screen device support for .NET Multi-platform App UI (.NET MAUI) apps.</Description>
+    <Description>Foldable or multi-screen device support for .NET Multi-platform App UI (.NET MAUI) apps. It is a for of MAUI with support for GTK.</Description>
```

Comment:
Typo: for -> fork.

But let's rather have this description: (MALI is a fork of Maui with support for GTK) Foldable or multi-screen device support for .NET Multi-platform App UI (.NET MAUI) apps.

Reply:
done